internal/repl/repl.go

```diff
@@ -151,9 +156,12 @@ func goRun(files []string) ([]byte, bytes.Buffer, error) {
 	cmd.Stdin = os.Stdin
 	cmd.Stderr = &stderr
 	out, err := cmd.Output()
+
 	return out, stderr, err
 }
 
+// If a valid expression, wraps the expression in a print statement to display
```

Comment:
doc-comments for a function usually look like:

```go
// <func-name> <verb> [etc...]
func <func-name>(...)
```

also, as it stands now, it seems like the sentence is missing one or two words.